tensorrt_llm/_torch/memory/gpu_memory_backend.py

```
@@ -0,0 +1,439 @@
+            )
+            self._client = None
+            return False
+
+        self._is_rw = self._client.granted_lock_type == GrantedLockType.RW
+
+        # GMS-backed VMM tensors can segfault if torch.cuda.empty_cache()
+        # tries to free them. Apply the upstream safety patch once we're
+        # connected. Idempotent and cheap.
+        try:
+            patch_empty_cache()
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)